### PR TITLE
[KO] translate site/ko/alpha/tutorials/quickstart

### PR DESCRIPTION
**Pull request by rickiepark**
Translate advanced.ipynb & beginner.ipynb
Colab check is done.
Please review it! :)

### CODE REVIEW

File: site/ko/alpha/tutorials/quickstart/advanced.ipynb [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+    "EPOCHS = 5\n",
+    "\n",
+    "for epoch in range(EPOCHS):\n",
+    "  for images, labels in train_ds:\n",
+    "    train_step(images, labels)\n",
+    "\n",
+    "  for test_images, test_labels in test_ds:\n",
+    "    test_step(test_images, test_labels)\n",
+    "\n",
+    "  template = 'Epoch {}, Loss: {}, Accuracy: {}, Test Loss: {}, Test Accuracy: {}'\n",
```
Comment (Choiuijin1125) [inline site/ko/alpha/tutorials/quickstart/advanced.ipynb]: There is missing colon, Epoch {} → Epoch: {}
And I'd like to suggest to translate this line :)

Comment (rickiepark) [reply site/ko/alpha/tutorials/quickstart/advanced.ipynb]: Good catch! :)

**Comment by rickiepark** [comment]
@Choiuijin1125 Can you review this PR? :)

**Comment by Choiuijin1125** [comment]
Sure! LGTM 👍

**Pull request merged**